addons/test_mail/models/test_mail_models.py

```diff
@@ -198,6 +207,18 @@ def _notify_get_recipients_groups(self, message, model_description, msg_vals=Non
 
         return groups
 
+    def _track_filter_displayed_tracking_values(self, tracking_values):
```

Comment:
This model is used in performance tests (via 'mail.test.ticket.mc') ... would try to avoid modifying it and keep the same behavior if possible, it allows to compare counters across versions as behavior is always the same ^^ . Do you need those changes ?

Reply:
Think these were mainly 'needed' for the _is_empty test, which we're removing anyway :)